cli/internal/cmd/minidown.go

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright (c) Edgeless Systems GmbH
+
+SPDX-License-Identifier: AGPL-3.0-only
+*/
+
+package cmd
+
+import (
+	"os"
+
+	"github.com/edgelesssys/constellation/v2/internal/constants"
+	"github.com/spf13/cobra"
+	"go.uber.org/multierr"
+)
+
+func newMiniDownCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "down",
+		Short: "Destroy a mini Constellation cluster",
+		Long:  "Destroy a mini Constellation cluster.",
+		Args:  cobra.ExactArgs(0),
+		RunE:  runDown,
+	}
+
+	return cmd
+}
+
+func runDown(cmd *cobra.Command, args []string) error {
+	err := runTerminate(cmd, args)
+	if removeErr := os.Remove(constants.MasterSecretFilename); removeErr != nil && !os.IsNotExist(removeErr) {
```

Comment:
Be careful: Don't remove the master secret unless we know it's actually from a Mini Constellation. Running `./constellation mini down` in an already existing workspace will nuke the master secret.

Reply:
First of, running `constellation mini down` in the wrong working directory is mostly user error, your OS also doesn't prevent you from running `rm -rf ./*` because you are in the "wrong" directory.

Currently, we don't have a way an easy way to check what kind of cluster the work space is configured for.
We could add additional "safeguards" by first checking if the statefile matches the QEMU provider and the name is set to `mini`